Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'rack-cors'
 gem 'active_model_serializers'
 gem 'devise_token_auth'
 gem 'pundit', '~> 1.1'
+gem 'stripe-rails'
+gem 'stripe-ruby-mock', '~> 2.5', require: 'stripe_mock'
```

Comment:
Please move this out of the production section. This gem is only used in development and test

Reply:
So stripe ruby mock in development test and stripe-rails in development then?

Reply:
Done.